src/plot_api/helpers.js

```diff
@@ -12,15 +12,15 @@
 var isNumeric = require('fast-isnumeric');
 var m4FromQuat = require('gl-mat4/fromQuat');
 
-var Registry = require('../registry');
 var Lib = require('../lib');
-var Plots = require('../plots/plots');
-var AxisIds = require('../plots/cartesian/axis_ids');
+var subplotsRegistry = require('../plots/plots').subplotsRegistry;
```

Comment:
The only function used from `Plots` in this file is `subplotsRegistry`.

Reply:
Is there a way to enforce this _rule_ using eslint? If not, than we don't _need_ to do this.